lib/msal-react-native-android-poc/SampleApp/index.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * @format
+ */
+
+import {AppRegistry} from 'react-native';
```

Comment:
This is just a nitpick, but for the sake of consistency I'd suggest adding spaces after opening and before closing curly braces.